extra-utils/wallpaperctl/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=wallpaperctl
+PKGSEC=utils
+PKGDEP="bash x11-app feh grep sed"
+BUILDDEPONLY=1
```

Comment:
Variable not yet implemented in Autobuild3. Please remove this line.

Reply:
Updated.